generate-stats

```diff
@@ -26,13 +26,14 @@ import logging
 import json
 from collections import Counter
 
-# Seconds in two weeks
-TWO_WEEKS = 1209600
+# Seconds in a day
+DAY = 86400
 
 
 def main():
     parser = argparse.ArgumentParser(description='Generate statistics about failed tests')
     parser.add_argument("--db", default="cockpit-failures.db", help="Database name")
+    parser.add_argument("--days", default=14, type=int, help="Number of days to take statistics from. Default is 14.")
```

Comment:
Nitpick: `Default is %(default)s.`

Reply:
oh nice! Didn't know about this, updated